nva-datamodel-java/src/test/java/no/unit/nva/MigrateSeriesIdTest.java

```diff
@@ -0,0 +1,38 @@
+package no.unit.nva;
+
+import static no.unit.nva.utils.MigratePublicationChannelIdTestUtils.SERIES_PATH_ELEMENT;
+import static no.unit.nva.utils.MigratePublicationChannelIdTestUtils.constructPublicationChannelId;
+import static no.unit.nva.utils.MigratePublicationChannelIdTestUtils.getNewIdentifierByOldIdentifier;
+import static no.unit.nva.utils.MigratePublicationChannelIdTestUtils.randomOldSeriesIdentifier;
+import static no.unit.nva.utils.MigratePublicationChannelIdTestUtils.randomYear;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import no.unit.nva.model.contexttypes.Series;
+import no.unit.nva.model.contexttypes.utils.ChannelType;
+import org.junit.jupiter.api.Test;
+
+class MigrateSeriesIdTest {
+
+    private static final ChannelType SERIES = ChannelType.SERIES;
+    private static final String OLD_IDENTIFIER_WITHOUT_MAPPING = "111111";
+
+    @Test
+    void shouldReplaceOldSeriesIdWithNewId() {
```

Comment:
The three test classes can be reduced to a single migration test that is parameterised and marked as deprecated. The code for this migration should also be marked as deprecated.

Reply:
I did not manage to reduce the tests to a single migration test because i use the constructors for Journal, Publisher and Series in the tests.
Do you have an example of how you would do this?